flash/image/segmentation/transforms.py

```diff
@@ -74,3 +76,44 @@ def predict_default_transforms(image_size: Tuple[int, int]) -> Dict[str, Callabl
         ),
         "collate": kornia_collate,
     }
+
+
+def remove_extra_dimensions(batch: Dict[str, Any]):
```

Comment:
@ethanwharris Not sure 100% why the list is generated. Seems to be coming from the kornia_collate.

Reply:
Hmm, maybe it's an issue with the `ApplyToKeys` maybe worth to check that the list is length 1 just to make sure we don't lose any data

Reply:
Yes, it is of size 1